pattern/constraint/type/OwnsConstraint.java

```diff
@@ -77,14 +83,18 @@ public Optional<TypeVariable> overridden() {
         return Optional.ofNullable(overriddenAttributeType);
     }
 
+    public TypeEdge.UniquenessConstraint uniqueness() {
+        return uniqueness;
+    }
+
     public boolean isKey() {
-        return isKey;
+        return uniqueness == TypeEdge.UniquenessConstraint.KEY;
     }
```

Comment:
This method should no longer be useful. If it is still used, you'll want to trace where and update that code based on the new `UniquenessConstraint`.